startupscript/post-startup.sh

```diff
@@ -74,8 +74,8 @@ cd /tmp || exit
 # Make the .workbench directory as the user so that they own it and have correct linux permissions.
 ${RUN_AS_LOGIN_USER} "mkdir -p '${USER_WORKBENCH_CONFIG_DIR}'"
 ${RUN_AS_LOGIN_USER} "ln -sf '${USER_WORKBENCH_CONFIG_DIR}' '${USER_WORKBENCH_LEGACY_CONFIG_DIR}'"
-exec >> "${POST_STARTUP_OUTPUT_FILE}"
-exec 2>&1
+exec > >(tee -a "${POST_STARTUP_OUTPUT_FILE}")  # Append output to the file and print to terminal
+exec 2> >(tee -a "${POST_STARTUP_OUTPUT_FILE}" >&2)  # Append errors to the file and print to terminal
```

Comment:
I think we should split stdout/stderr to echo as well, I'm tired of looking at post-startup-output.txt every single time. This should give us better logs both for the smoke tests here, and when looking at the journalctl logs when iterating on a devcontainer.

Reply:
agree. sgtm